doc/build-openbsd.md

```diff
@@ -4,67 +4,24 @@ OpenBSD build guide
 
 This guide describes how to build bitcoind and command-line utilities on OpenBSD.
 
-As OpenBSD is most common as a server OS, we will not bother with the GUI.
+OpenBSD is most commonly used as a server OS, so this guide does not contain instructions for building the GUI.
 
 Preparation
 -------------
 
 Run the following as root to install the base dependencies for building:
 
 ```bash
-pkg_add gmake libtool libevent
+pkg_add g++ git gmake libevent libtool
 pkg_add autoconf # (select highest version, e.g. 2.69)
 pkg_add automake # (select highest version, e.g. 1.15)
-pkg_add python # (select highest version, e.g. 3.5)
-```
-
-See [dependencies.md](dependencies.md) for a complete overview.
-
-GCC
--------
-
-The default C++ compiler that comes with OpenBSD 5.9 is g++ 4.2. This version is old (from 2007), and is not able to compile the current version of Bitcoin Core, primarily as it has no C++11 support, but even before there were issues. So here we will be installing a newer compiler:
```

Comment:
Not sure why you're removing the explicit description for GCC here - it's intentional that there is a "GCC" and "Clang" header, with the instructions above shared between both.

Reply:
Overzealous deleting. Fixed.